src/convertToObject.js

```diff
@@ -12,7 +12,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const string = sourceString.split(';');
+
+  return string.reduce((acc, item) => {
+    const rule = item.split(':');
```

Comment:
Можеш погратися з деструктуризацією
```suggestion
    const [key, value] = item.split(':');
```